### PR TITLE
Setup Wizard: Support code host connection edit UI

### PR DESCRIPTION
**Pull request by vovakulikov**
Closes https://github.com/sourcegraph/sourcegraph/issues/47237

This PR connects code host connection edit UI to edit *update code host mutation and also connects a proper access token validation logic.

## Test plan
- You should be able to create new code host connection 
- Make sure that code host access token is validated (you can't submit form if token is invalid)
- Make sure that you can pick repositories or/and organisations through pickers UI
- Make sure that can edit code host connection 

## App preview:

- [Web](https://sg-web-vk-support-code-host-edit-in.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/web/src/setup-wizard/components/remote-repositories-step/components/code-hosts/github/GithubConnectView.tsx [hunk 1]
```diff
@@ -22,13 +23,23 @@ import {
...
+    ExternalServiceCheckConnectionByIdResult,
+    ExternalServiceCheckConnectionByIdVariables,
+    ExternalServiceKind,
+    ValidateAccessTokenResult,
+    ValidateAccessTokenVariables,
+} from '../../../../../../graphql-operations'
 import { CodeHostJSONFormContent, RadioGroupSection, CodeHostConnectFormFields, CodeHostJSONFormState } from '../common'
 
 import { GithubOrganizationsPicker, GithubRepositoriesPicker } from './GithubEntityPickers'
+import { getAccessTokenValue, getRepositoriesSettings } from './helpers'
```
Comment (st0nebraker) [inline client/web/src/setup-wizard/components/remote-repositories-step/components/code-hosts/github/GithubConnectView.tsx:42]: This import is failing locally for me. Did you forget to save a file? Or are these functions renamed and somewhere else? @vovakulikov

Comment (vovakulikov) [reply client/web/src/setup-wizard/components/remote-repositories-step/components/code-hosts/github/GithubConnectView.tsx:42]: Oh, yes It was included in the commit, sorry. I just pushed it

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.01% (+1.89 kb) | 0.02% (+1.89 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/vk/support-code-host-edit-in-wizard/compare-report.html;tab=live_object) for a full comparison between the commits f3cb48d and 745a478 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/setup-wizard/components/remote-repositories-step/components/code-hosts/github/GithubConnectView.tsx (hunk 1 above)
Comment (st0nebraker) [reply client/web/src/setup-wizard/components/remote-repositories-step/components/code-hosts/github/GithubConnectView.tsx:42]: Thanks! I see it now

**Review by st0nebraker — APPROVED** [review]
Looks good to me! I'm seeing the delete and connect action take place or block if access token is wrong. Caveat that I couldn't test happy path for org's yet, but it seems more like a BE problem not impacted by this PR. I've flagged this to Gary to look at. Repo picker and edit mode worked great though. 🙌🏻

**Pull request merged**